tests/helpers.py

```diff
@@ -1,89 +1,45 @@
-import json
+from contextlib import contextmanager
 
-from deepdiff import DeepDiff
+from db import db
+from sqlalchemy import event
+from sqlalchemy import func
+from sqlalchemy.engine import Engine
+from sqlalchemy.inspection import inspect
 
 
-def expected_data_within_get_response(
-    test_client,
-    endpoint: str,
-    expected_data,
-    exclude_paths=None,
-    exclude_regex_paths=None,
-):
-    """
-    Given a endpoint and expected content,
-    check to see if response contains expected data
-
-    Args:
-        test_client: A flask test client
-        endpoint (str): The GET request endpoint
-        expected_data: The content we expect to find
-        exclude_paths: DeepDiff dict paths to exclude from diff errors
-        exclude_regex_paths: DeepDiff dict regex paths
-            to exclude from diff errors
-
-    """
-    response = test_client.get(endpoint, follow_redirects=True)
-    response_data = json.loads(response.data)
-
-    diff = DeepDiff(
-        expected_data,
-        response_data,
-        exclude_paths=exclude_paths,
-        exclude_regex_paths=exclude_regex_paths,
-    )
+@contextmanager
+def no_gather_sql():
+    @event.listens_for(Engine, "before_cursor_execute", retval=True)
```

Comment:
whats the purpose of this?

Reply:
We don't want to time sql queries which are executed as test helpers, because they don't reflect sql which will run in production.

Reply:
This context manager adds a comment to those queries which makes the timer ignore them.